USERS-GUIDE.md

```diff
@@ -133,7 +133,7 @@ _Type converters_ which implement `TypeConverter` interface are responsible for
 
 The _conf4j_ library provides type converters for all primitive types (like `boolean`, `int`, `double`),
 primitive wrapper types (like `Boolean`, `Integer`, `Double`), _enumerations_ and many other types which are frequently
-used (like `BigDecimal`). More complex types are also supported - there is a dedicated converter for `List<E>` and `Map<K, V>`
+used (like `BigDecimal`, `LocalDateTime`). More complex types are also supported - there is a dedicated converter for `List<E>` and `Map<K, V>`
```

Comment:
I don't think there is a need to list in the documentation every single converter. We may create a list of table with converters somewhere in this section in the future, but for now let's not update this section.